src/main/java/com/Main.java

```diff
@@ -0,0 +1,63 @@
+package com;
+import com.duke.Duke;
+import com.task.*;
+
+import java.util.Scanner;
+
+
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        String line;
+        Duke duke = new Duke();
+        //duke.greet();
+        do {
+            line = in.nextLine();
+            line = line.toLowerCase();
+            switch(line) {
+            case "bye":
```

Comment:
If `bye` is  a fallthrough case, then a separate comment `//Fallthrough` needs to be included, like in this [switch-case layout](https://se-education.org/guides/conventions/java/basic.html#layout).